src/plugins/data/public/antlr/opensearch_ppl/grammar/OpenSearchPPLParser.g4

```diff
@@ -263,9 +263,9 @@ comparisonExpression
    ;
 
 valueExpression
-   // : left = valueExpression binaryOperator = (STAR | DIVIDE | MODULE) right = valueExpression   # binaryArithmetic
-   // | left = valueExpression binaryOperator = (PLUS | MINUS) right = valueExpression             # binaryArithmetic
-   : primaryExpression                                                                          # valueExpressionDefault
+   : left = valueExpression binaryOperator = (STAR | DIVIDE | MODULE) right = valueExpression   # binaryArithmetic
```

Comment:
NOTE: this is reverting back a change made in a previous pr, to go back to the original grammar.